themes/default/content/blog/review-stacks/index.md

```diff
@@ -0,0 +1,97 @@
+
+> “Pulumi allows every engineer to build and update infrastructure instead of only our infrastructure team, which enables us to ship faster than if we were using a domain-specific language,” said Aaron Gibralter, head of engineering, Mighty. “With Pulumi Deployments, we could easily set up a standardized CI/CD infrastructure workflow for every engineer that immediately makes them productive. Review Stacks automates the work of managing our own ephemeral infrastructure and makes our lives easier.”
+
+Review Stacks also provide an avenue for optimizing cloud costs. Rather than keeping development infrastructure running 24/7, Review Stacks only exist as long as an engineer works on a feature. Since review stacks are flexible and built on Pulumi IaC, it is also possible to configure Review Stacks to deploy to multi-tenant environments that share development infrastructure to further streamline costs.
+
+> “At GitHub, we are constantly seeking to improve the developer experience and our partner ecosystem plays a big role in that.” said Matthew Manning, Senior Business Development Manager for GitHub Actions, “We’re excited to support the launch of Pulumi Review Stacks with GitHub because it makes it easier for teams to review and test their products before shipping.”
+
+## Configuring Review Stacks
+
+Configuring Review Stacks is a simple three-step process:
+
+1. Create a new stack and `Pulumi.<stack>.yaml` configuration file - this config will be copied into every review stack that gets created, and can even be modified within a PR
+2. Configure Deployment Settings for the stack - this specifies how to acquire source code, cloud credentials and more when deploying via Pulumi Deployments
+3. Set the `pullRequestTemplate` Deployment Setting to true - this indicates that all pull requests against this stack’s branch should reference this stack as a Review Stack template.
+
+You can use an existing stack as a Review Stack template, as long as it has Pulumi Deployment settings configured. This will cause Review Stacks to get deployed into the same cloud account. If you want to separate the cloud resources in your production stack from the resources created via Review Stacks then you can create a separate stack and template that references a different cloud account (AWS, Azure, GCP, etc).
+
+Review Stacks and Deployment Settings can be configured via the Pulumi Cloud console, the Pulumi Cloud REST API, or within a Pulumi Program using the Pulumi Cloud Resource Provider.
+
+### REST API
```

Comment:
It seems we're missing the "Pulumi Cloud console" option as a drill down?  Might provider opportunity to have a screenshot of this?

Reply:
Yup, will add the screenshot. Code changes had not been deployed when I wrote the draft.